swap_meet/vendor.py

```diff
@@ -1,2 +1,108 @@
+from typing import ItemsView
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
```

Comment:
The `else` is unneeded here because the `if` block returns if it's entered. We could instead write:

```
def remove(self, item):
        if item in self.inventory:
            self.inventory.remove(item)
            return item
        return False
```